src/util/std_expr.h

```diff
@@ -4750,4 +4750,49 @@ inline void validate_expr(const popcount_exprt &value)
   validate_operands(value, 1, "popcount must have one operand");
 }
 
+/// this is a parametric version of an if-expression: it returns
+/// the value of the first case (using the ordering of the operands)
+/// whose condition evaluates to true.
+class cond_exprt : public multi_ary_exprt
```

Comment:
Seems easily confused with `if_exprt`, how about something like `nary_if_exprt` or `if_elseif_exprt`?

Reply:
Agreed that something like that would be better; however, ID_cond is the existing name, and renaming should be a separate job. The added class simply documents the current state.